packages/vega-transforms/src/Aggregate.js

```diff
@@ -11,6 +11,7 @@ import {accessorFields, accessorName, array, error, inherits} from 'vega-util';
  * @param {Array<function(object): *>} [params.groupby] - An array of accessors to groupby.
  * @param {Array<function(object): *>} [params.fields] - An array of accessors to aggregate.
  * @param {Array<string>} [params.ops] - An array of strings indicating aggregation operations.
+ * @param {Array<object>} [params.aggregate_params=[null]] - An optional array of parameters for aggregation operations.
```

Comment:
Do we need to repeat the name `aggregate` here or is it already obvious because of the context of the aggregate?

Reply:
I was going by the naming agreed upon in https://github.com/vega/vega-lite/issues/4439. We cannot just call it params.params, as this would conflict with the params argument in windowing functions. Anytime you would want to apply an aggregate with parameter to over a window with a parameter, you would run into problems.